README.md

```diff
@@ -10,35 +10,97 @@ Compile, edit namelist, and run canopy model:
+    | Variable Name    | Variable Descrip. (Units)                         |
+    | ---------------  | ------------------------------------------------- |  
+    | LAT              | Latitude  (degrees)                               |
+    | LON              | Longitude (degrees)                               |
+    | FH               | Forest canopy height (m)                          |
+    | WS               | Wind speed at reference height (m/s), e.g., 10 m  |
+    | CLU              | Forest clumping index (dimensionless)             |
+    | LAI              | Leaf area index (m2/m2)                           |
+    | VTYPE            | Vegetation type (dimensionless), e.g., VIIRS      |
+    | FFRAC            | Forest fraction (dimensionless)                   |
+    | UST              | Friction velocity (m/s)                           |
+    | CSZ              | Cosine of the solar zenith angle (dimensionless)  |
+    | Z0               | Total surface roughness length (m)                |
+    | MOL              | Monin-Obukhov Length (m)                          |
+    | FRP              | Total Fire Radiative Power (MW/grid cell)         |
 
+2.  Generic 1D canopy profile specifications (e.g., `input_profile.txt`).
 
-Citations:
+    Namelist Option : `file_prof`
+
+    | Variable Name    | Variable Descrip. (Units)                         |
+    | ---------------  | ------------------------------------------------- |
+    | CANLAY           | Canopy layer number (integer)                     |
+    | ZK               | Canopy layer resolution (m)                       |
+
+
+    Current User Namelist Options:
+
+    | Variable Name    | Variable Descrip. (Units)                                                          |
+    | ---------------  | ---------------------------------------------------------------------------------- |
+    | nlat             | integer number of latitude cells (must match # of LAT in `file_vars` above )       |
+    | nlon             | integer number of longitude cells (must match # of LON in `file_vars`above )       |
+    | canlays          | integer number of canopy layers  (must match # of CANLAY in `file_prof` above )    |
+    | canres           | real value of canopy vertical resolution (m) (must match ZK in `file_prof` above ) |
+    | ifcanwind        | logical canopy wind/WAF option (default = .FALSE.)                                 |
+    | ifcaneddy        | logical canopy eddy Kz option (default = .FALSE.)                                  |
+    | ifcanphot        | logical canopy photolysis option (default = .FALSE.)                               |
+    | href             | real value of reference height above canopy associated with input wind speed (m)   |
+    | z0ghcm           | ratio of ground roughness length to canopy top height (Massman et al., 2017)       |
+    | lamdars          | Influence function associated with roughness sublayer (Massman et al., 2017)       |
```

Comment:
"influence function"? seems like it's just a single number

Reply:
(apologies if I commented on this elsewhere)

Reply:
Thanks Zach.   Yes, this is not a good description.  I have updated it in the next [commit](https://github.com/noaa-oar-arl/canopy-app/pull/20/commits/1d7e925674e0c389b6f21d55eaeb6b468e49796f).

Reply:
:+1: